### PR TITLE
Add UseRdonly option to the airbyte connector

### PR DESCRIPTION
**Pull request by Anders-PlanetScale**
An attempt to add a Use Rdonly option to the airbyte connector. I am not a go person so someone please review.

### CODE REVIEW

**Comment by Anders-PlanetScale** [comment]
@notfelineit any chance we can get this reviewed and merged?

**Comment by notfelineit** [comment]
@Anders-PlanetScale sorry for the delay - I'll review today!

File: cmd/airbyte-source/spec.json [hunk 1]
```diff
@@ -52,12 +52,19 @@
           "default": false,
           "order": 4
         },
+        "use_rdonly": {
+          "description": "Use an rdonly replica to pull data from",
```
Comment (notfelineit) [inline cmd/airbyte-source/spec.json:56]:
```suggestion
          "description": "Pull data from a read-only replica",
```
super nit, sorry!

File: cmd/internal/planetscale_edge_database.go [hunk 2]
```diff
@@ -171,7 +171,9 @@ func (p PlanetScaleEdgeDatabase) Read(ctx context.Context, w io.Writer, ps Plane
 	)
 
 	tabletType := psdbconnect.TabletType_primary
-	if ps.UseReplica {
+	if ps.UseRdonly {
+		tabletType = psdbconnect.TabletType_batch
```
Comment (notfelineit) [inline cmd/internal/planetscale_edge_database.go:175]: Confirmed that this will be mapped to `topodatapb.TabletType_RDONLY` by our edge gateway.

**Review by notfelineit — APPROVED** [review]

**Pull request merged**